package.json

```diff
@@ -7,29 +7,29 @@
     "start": "ng serve",
     "serve:ssr": "node dist/server.js",
     "build": "npm run clean && npm run build:app && npm run build:server",
-    "build:ghpages": "npm run clean && npm run build:app -- --base-href '/ngx-drag-to-select/' && npm run build:server",
+    "build:ghpages": "npm run clean && npm run build:app -- --base-href /ngx-drag-to-select/ && npm run build:server",
     "build:ssr": "npm run build && npm run webpack:server",
     "build:prerender": "npm run build:ssr && npm run generate:prerender",
     "build:prerender-ghpages": "npm run build:ghpages && npm run webpack:server && npm run generate:prerender",
-    "build:app": "ng build --prod",
-    "build:server": "ng build --prod --app 1 --output-hashing=false",
-    "build:lib": "npm run clean && npm run packagr && npm run build:lib:sass && npm run copy:styles && npm run copy:readme",
-    "build:lib:sass": "node-sass src/lib/scss/ngx-drag-to-select.scss -o dist/lib",
-    "packagr": "ng-packagr -p src/lib/package.json",
+    "build:app": "ng build --prod --project=app",
+    "build:server": "ng build --prod --project=server",
+    "build:lib": "npm run clean && npm run build:lib:code && npm run build:lib:sass && npm run copy:styles && npm run copy:readme",
+    "build:lib:code": "ng build --prod --project=ngx-drag-to-select",
+    "build:lib:sass": "node-sass projects/ngx-drag-to-select/src/scss/ngx-drag-to-select.scss -o dist/ngx-drag-to-select",
     "deploy": "ngh --dir dist/browser",
-    "copy:styles": "cpr src/lib/scss dist/lib/scss",
-    "copy:readme": "copyfiles README.md dist/lib",
+    "copy:styles": "cpr projects/ngx-drag-to-select/src/scss dist/ngx-drag-to-select/scss",
+    "copy:readme": "copyfiles README.md dist/ngx-drag-to-select",
     "generate:prerender": "node dist/prerender.js",
     "webpack:server": "webpack --config webpack.server.config.js --progress --colors",
-    "format:base": "prettier \"{src,cypress}/**/*{.ts,.js,.json}\"",
+    "format:base": "prettier \"{src,cypress,projects}/**/*{.ts,.js,.json}\"",
     "format:check": "npm run format:base -- --list-different",
     "format:fix": "npm run format:base -- --write",
     "style": "npm run lint:check && npm run format:check",
     "style:fix": "npm run lint:fix && npm run format:fix",
     "clean": "del dist",
-    "test": "jest",
+    "test": "jest --config ./projects/ngx-drag-to-select/jest.config.js",
     "test:ci": "npm run test && start-server-and-test start http-get://localhost:4200 e2e:ci",
-    "test:watch": "jest --watch",
+    "test:watch": "npm run test -- --watch",
     "lint:check": "ng lint",
     "lint:fix": "ng lint --fix",
```

Comment:
This command is broken with the latest CLI. It doesn't allow multiple apps for a single command I suppose. We have to change this to `ng lint app --fix && ng lint ngx-drag-to-select --fix`.

Reply:
Ah good catch, you're right.

Reply:
Done!

Reply:
Do you need the `--project` flag?

Reply:
Nope we don't! I also removed the `--project` flag in the build scripts.

Reply:
Yay 🎉